lib/modulr.rb

```diff
@@ -3,17 +3,6 @@
 require_relative "modulr/version"
 
 require_relative "modulr/auth/signature"
-
```

Comment:
I cleaned a little bit how we required different files in modulr dir

Reply:
None will be used more than once than the required ones that you have taken from the file?

Reply:
For gems, it's pretty common to centralize the requirements in the main lib file, but I'm ok with dispersing if the rest of the team finds it more precise and more maintainable.